crashlytics/src/AndroidImpl.cs

```diff
@@ -141,6 +142,39 @@ internal class AndroidImpl : Impl {
         }, "LogException");
     }
 
+    public override void LogExceptionAsFatal(Exception exception) {
+      var loggedException = LoggedException.FromException(exception);
+      Dictionary<string, string>[] parsedStackTrace = loggedException.ParsedStackTrace;
+      
+      if (parsedStackTrace.Length == 0) {
```

Comment:
Hey @cynthiajoan, we have addressed the TODO previously we had. If the stack trace from exception is empty, we will parse the current stack trace in with two frames less to prevent false blame on  crashlytics sdk. Do you mind check this logic and lgtm again, thank you!